Source/XR-Interaction-Component/Source/Runtime/Interaction/Interactables/TeleportationAnchorProximityEntry.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+using VRBuilder.Core.Properties;
+
+namespace VRBuilder.XRInteraction
+{
+    /// <summary>
+    /// This adds the possibility to move the user into a <seealso cref="TeleportationAnchor"/> and trigger the teleport event without teleporting.
+    /// It will initiate the next step in the VR-Builder process.
```

Comment:
I'd remove this line as it's not what the class does - just a consequence of it.
```suggestion
```